examples/00-fluent/modeling_cavitation.py

```diff
@@ -0,0 +1,377 @@
+"""
+
+Modeling Cavitation
+-------------------
+This example examines the pressure-driven cavitating flow of water
+through a sharp-edged orifice. This is a typical configuration in
+fuel injectors, and brings a challenge to the physics and numerics
+of cavitation models because of the high pressure differentials
+involved and the high ratio of liquid to vapor density.
+
+This example uses the multiphase modeling capability of Ansys
+Fluent, and will be able to predict the strong cavitation near the
+orifice after flow separation at a sharp edge.
+
+**Workflow tasks**
+
+This example demonstrates how to do the following:
+
+- Set boundary conditions for internal flow
+- Use the mixture model with cavitation effects
+- Calculate a solution using the pressure-based coupled solver
+
+**Problem description**
+
+The problem considers the cavitation caused by the flow separation
+after a sharp-edged orifice. The flow is pressure driven, with an
+inlet pressure of 5 x 10^5 Pa and an outlet pressure of 9.5 x 10^4 Pa.
+The orifice diameter is 4 x 10^-3 m, and the geometrical parameters
+of the orifice are D/d = 2.88 and L/d = 4, where D, d, and L are the
+inlet diameter, orifice diameter, and orifice length respectively.
+"""
+
+###############################################################################
+# Example Setup
+# -------------
+# Before you can begin, you must set up the example and initialize this workflow.
+#
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports, which includes downloading and importing
+# the geometry file.
+
+# sphinx_gallery_thumbnail_path = '_static\cavitation_model.png'
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core import examples
+
+cav_file = examples.download_file(
+    "cav.msh", "pyfluent/cavitation", save_path=pyfluent.EXAMPLES_PATH
+)
+
+###############################################################################
+# Launch Fluent
+# ~~~~~~~~~~~~~
+# Launch Fluent as a service in 2d solution mode with double precision running
+# on one processor.
+
+solver = pyfluent.launch_fluent(
```

Comment:
Staring at the documentation build error from the last pipeline execution I think you might need to force use of 24.1 here.

Reply:
You are correct, already mentioned this to Adam: we are using the last publicly released Fluent version to run all the examples. It might be possible to find equivalent commands that work in 23.2 so the example can be released now, or can wait until 24.1 is publicly released

Reply:
Yeah, I'm kind of guessing that's what might be the issue.  There is also this bit in the documentation build output:
```
pyfluent.launcher WARNING: These specified arguments are only supported when starting local standalone Fluent clients: cwd.
```

Reply:
That is a somewhat expected warning, in the sense that contributors usually test using Windows and local standalone Fluent clients, whereas in GitHub the tests run using Linux and inside containers. 

Simply removing `cwd` specification here will make the warning go away and otherwise make no difference in the GitHub examples gallery run, but will make a difference when running the examples locally using Windows/standalone Fluent.

We made changes (mainly myself and Mainak) in past PRs to try to abstract these differences away and make PyFluent work as similarly as possible and independent of the system and Fluent setup, as far as GitHub tests and examples are concerned. But there are still remaining differences, and this is one of them as you can tell by the warning message. 

Alternatively, we could add support for the `launch_fluent(cwd=...)` argument in Fluent container launches, perhaps using a `pyfluent.examples_path` object that behaves differently depending on how Fluent is being launched; or redirecting `cwd` to specify the host path that is being mounted into the guest container (though I am not sure if this is very intuitive, and will also muddy the `cwd` argument a bit)

Reply:
OK I think we should leave it out and have no warnings in the build pipeline.  The `cwd` arg is not necessarily required in local standalone runs either right and someone could always add it locally to get things working.

Reply:
That is correct. I believe the only reason to keep `cwd` in the examples would be to demonstrate a very commonly used/useful command to folks learning PyFluent.

Regardless, leaving it out is fine by me.